backend/models/core/Entity.js

```diff
@@ -122,6 +122,90 @@ class Entity {
         })); 
   }
 
+  static async findByPageAndEntry({entity_type_slug , entry, page }) {
+    const db = new DB();
+     
+    let totalRows;
+    let totalOrders;
+    
+    const totalRowsSQL = `select COUNT(entities.id) 
+                          from entity_types LEFT JOIN entities ON entities.entity_type_id = entity_types.id 
+                          WHERE entity_types.slug = :entity_type_slug;
+                          `;
+
+    const totalRowsData = await db.executeStatement(totalRowsSQL,[
+      {name: 'entity_type_slug', value:{stringValue: entity_type_slug}},
+    ]);
+
+    [{longValue:totalRows}] = totalRowsData.records[0]
+
+    const totalOrdersSQL = `select COUNT(attributes.order)
```

Comment:
Capitalize all predefined SQL statements (e.g. SELECT, INSERT, UPDATE etc)
```suggestion
    const totalOrdersSQL = `SELECT COUNT(attributes.order)
```